paket.dependencies

```diff
@@ -1,8 +1,9 @@
 source https://nuget.org/api/v2
 
 nuget FSharp.Formatting
-nuget NUnit 2.6.3
-nuget NUnit.Runners 2.6.3
+nuget Newtonsoft.Json
```

Comment:
Why do we need the dependency to `Newtonsoft.Json`?
I think that `FSharp.Data` has everything for JSON manipulation. 
https://github.com/sergey-tihon/SwaggerProvider/blob/master/paket.dependencies#L18-L20